### PR TITLE
ci: add GH Actions dependabot

### PR DESCRIPTION
**Pull request by endersonmaia**
This PR will add a CI job to check for updates on actions used on the CI.

You can see an example PR here: https://github.com/sunodo/sunodo/pull/189

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -297,7 +303,3 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [0.4.0]: https://github.com/cartesi/rollups/releases/tag/v0.4.0
 [0.3.0]: https://github.com/cartesi/rollups/releases/tag/v0.3.0
 [0.2.0]: https://github.com/cartesi/rollups/releases/tag/v0.2.0
-[0.1.3]: https://github.com/cartesi/rollups/releases/tag/v0.1.3
-[0.1.2]: https://github.com/cartesi/rollups/releases/tag/v0.1.2
-[0.1.1]: https://github.com/cartesi/rollups/releases/tag/v0.1.1
-[0.1.0]: https://github.com/cartesi/rollups/releases/tag/v0.1.0
```
Comment (marcelstanley) [inline CHANGELOG.md:303]: Any reason for deleting the `Unreleased` and `pre-0.2.0` releases from the list?

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,8 +36,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed `advance-runner`, `dispatcher`, `graphql-server`, `host-runner`, `inspect-server`, `indexer`, and `state-server` docker images
 
 ### Fixed
+
```
Comment (marcelstanley) [inline CHANGELOG.md:39]: Linter execution detected. Should we add a GH action for that?

**Comment by gligneul** [comment]
For some reason, the CI is not working correctly for PR made from forks. I changed the base to a branch called [feature/add-gh-actions-dependabot](https://github.com/cartesi/rollups-node/tree/feature/add-gh-actions-dependabot) so we can merge there and check if the CI is passing before merging it to main.

@endersonmaia, I also added you as a writer for the repo; now, you should be able to create branches without the fork.

File: CHANGELOG.md (hunk 1 above)
Comment (endersonmaia) [reply CHANGELOG.md:303]: No reason, I'll put those back.

**Comment by endersonmaia** [comment]
> For some reason, the CI is not working correctly for PR made from forks. I changed the base to a branch called [feature/add-gh-actions-dependabot](https://github.com/cartesi/rollups-node/tree/feature/add-gh-actions-dependabot) so we can merge there and check if the CI is passing before merging it to main.
> 

If the CI uses some Secrets, they'll fail.

> @endersonmaia, I also added you as a writer for the repo; now, you should be able to create branches without the fork.

:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (endersonmaia) [reply CHANGELOG.md:303]: Actually, my VSCode is removing those when I save the file.

**Comment by endersonmaia** [comment]
moved to PR https://github.com/cartesi/rollups-node/pull/120

File: CHANGELOG.md (hunk 1 above)
Comment (endersonmaia) [reply CHANGELOG.md]: That was the reason: https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md053.md

**Pull request closed without merging**